### PR TITLE
revert dev changes to rpi 5.15.y drivers adding interlude audio code

### PR DESCRIPTION
**Pull request by Yash-BlueRockSoft**
closing out jira ticket BA-193 driver cleanup and dev changes revert

### CODE REVIEW

File: sound/soc/bcm/rpi-wm8804-soundcard.c [hunk 1]
```diff
@@ -73,8 +73,6 @@ static struct gpio_desc *led_gpio_3;
 static struct snd_soc_dai_link snd_allo_digione_dai[];
 static struct snd_soc_card snd_rpi_wm8804;
 
-static uint32_t sysclk_freq = 27000000;
```
Comment (benpayne) [inline sound/soc/bcm/rpi-wm8804-soundcard.c]: Why are you undoing this change.  This is a good change.

File: sound/soc/codecs/cs4271.c [hunk 2]
```diff
@@ -495,19 +488,12 @@ static struct snd_soc_dai_driver cs4271_dai = {
...
 	if (gpio_is_valid(cs4271->gpio_nreset)) {
-		pr_err("reset setup %d", cs4271->gpio_nreset);
 		gpio_direction_output(cs4271->gpio_nreset, 0);
-		gpio_set_value(cs4271->gpio_nreset, 0);
-		//gpio_direction_output(23, 0);
-		//gpio_set_value(23, 0);
 		mdelay(1);
 		gpio_set_value(cs4271->gpio_nreset, 1);
-		mdelay(10);
+		mdelay(1);
```
Comment (benpayne) [inline sound/soc/codecs/cs4271.c:495]: Did you test this change with the hat and had no issues?

**Review by benpayne — COMMENTED** [review]
See comments in code

File: sound/soc/codecs/cs4271.c (hunk 2 above)
Comment (Yash-BlueRockSoft) [reply sound/soc/codecs/cs4271.c:495]: yes change has been tested

**Comment by Yash-BlueRockSoft** [comment]
validated changes and builds completed successfully merging pull request

**Pull request merged**